projects/control-service/projects/job-builder/build_image.sh

```diff
@@ -14,6 +14,7 @@ git_repository=$7
 registry_type=$8
 registry_username=$9
 registry_password=${10}
+extra_auth=${11}
```

Comment:
We have multiple job-builder images. They all 3 are currently based on kaniko but kaniko is not necessary the only build system. I feel like the contact between what control service passes and the job-builder image accept should not rely on Control Service knowing the implementation of the job-builder image.